.github/workflows/tests.yml

```diff
@@ -101,6 +101,19 @@ jobs:
       - name: Make and run tests
         run: docker exec builder bash -c 'cd FlexibleSUSY && VERBOSE=1 make -j2 all-test'
 
+      - name: Testing pack-SM-src
+        run: |
+          docker exec builder bash -c 'cd FlexibleSUSY && make pack-SM-src'
+          docker exec builder bash -c 'cd FlexibleSUSY && rm -rf models/SM'
+          docker exec builder bash -c 'cd FlexibleSUSY && make pack-SM-src'
```

Comment:
I think this line (and the following one) will not work, because in L107 the model is removed.
Please remove L108 and L109.